components/runtime/wine/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2019 Evgeny Litvinenko
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		wine
+COMPONENT_VERSION=	4.4
+COMPONENT_REVISION=     1
+COMPONENT_PROJECT_URL=	https://www.winehq.org/
+COMPONENT_SUMMARY=	Wine - Run Windows applications.
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:db3a09cf4b0b4329ffed8014073c9868b5ecb4826adfb4a91aa49bd2841207b0
+COMPONENT_DOWNLOAD_URL= git://source.winehq.org/git/$(COMPONENT_NAME).git
+COMPONENT_ARCHIVE_URL=	https://dl.winehq.org/wine/source/4.x/$(COMPONENT_ARCHIVE)
+COMPONENT_SIG_URL=	$(COMPONENT_ARCHIVE_URL).sign
+COMPONENT_FMRI=		runtime/wine
+COMPONENT_LICENSE=	LGPLv2.1
+COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
+COMPONENT_CLASSIFICATION=	Applications/System Utilities
+COMPONENT_BUGDB=	utility/wine
+
+USERLAND_GIT_REMOTE=	https://github.com/OpenIndiana/oi-userland.git
```

Comment:
Unused pieces of code should be removed.

Reply:
fixed

Reply:
Still see USERLAND_GIT_REMOTE

Reply:
@pyhalov, without USERLAND_GIT_REMOTE gmake REQUIRED_PACKAGES gives an error:
pkgmogrify: File wine.p5m line 16: Transform specified in file /home/src/oi-userland/transforms/defaults, line 191 reports Malformed action at position: 40: missing value:
    set name=userland.info.git-remote value=
                                            ^

Reply:
That's strange, it should be set from git remote. 
Look at make-rules/ips-buildinfo.mk

Reply:
Works fine for me without `USERLAND_GIT_REMOTE` in the Makefile... Perhaps you have some local changes you did not push to the PR?

Reply:
Fixed (USERLAND_GIT_REMOTE removed).
The upstream was named 'upstream' instead of 'origin' in my copy of oi-userland repo as a result USERLAND_GIT_REMOTE was empty.